UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Program.cs

```diff
@@ -0,0 +1,27 @@
+WebApplicationBuilder? builder = WebApplication.CreateBuilder(args);
```

Comment:
Don't think this needs to be nullable:
```suggestion
WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
```

Reply:
Done. Thanks Barrie